src/components/QueryCondition.vue

```diff
@@ -5,10 +5,10 @@
 			:disabled="!canDelete"
 			@click="removeCondition"
 		/>
-		<div class="query-condition__toggle-button-group">
-			<Button disabled="true" style="border-right: 1px darkgrey solid;">with</Button>
-			<Button disabled="true">without</Button>
-		</div>
+		<NegationToggle
+			class="query-condition__toggle-button-group"
```

Comment:
We not using this class anywhere, right? If so, then we can just drop it

Reply:
It's used (at least for now, until we use the wikit component, look at `&__toggle-button-group` line 190.

Reply:
Narf! I totally missed that! Cool, then that class is fine right now. It's naming is not ideal, but that can be changed in another PR, not this one.